sites/header/_base.scss

```diff
@@ -0,0 +1,14 @@
+/*
+ * Site header
+ */
+
+#wb-bnr + hr {
+	border-top: 3px solid #38414d;
+	margin-bottom: 0px;
+	margin-top: 5px;
+
+	+ .container {
+		font-size: 20px;
+		padding-left: 0px;
```

Comment:
I'd also add padding-right: 0px; for XS small view:
```suggestion
		padding-right: 0px;
```
See the gap if this rule is not applied:
<img width="431" alt="Screen Shot 2022-05-17 at 5 08 00 PM" src="https://user-images.githubusercontent.com/4353199/168910406-96718c8e-5aa2-4baa-9aa7-23c56be6d433.png">